LeanplumTests/LPMultiApiTest.m

```diff
@@ -0,0 +1,182 @@
+//
+//  LPMultiApiTest.m
+//  LeanplumTests
+//
+//  Created by Mayank Sanganeria on 5/22/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <OHHTTPStubs/OHHTTPStubs.h>
+#import <OHHTTPStubs/OHPathHelpers.h>
+#import "LPMultiApi.h"
+#import "LPAPIConfig.h"
+#import "LPConstants.h"
+#import "LPTestHelper.h"
+
+@interface LPMultiApiTest : XCTestCase
+
+@end
+
+@implementation LPMultiApiTest
+
+- (void)setUp {
+    [super setUp];
+    [LPTestHelper setup];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    [OHHTTPStubs removeAllStubs];
+}
+
+- (void)testMultiApi {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    [LPMultiApi multiWithData:[self sampleData] parameters:nil success:^ {
+        [expectation fulfill];
+    } failure:^(NSError *error) {
+    }];
+    
+    [self waitForExpectationsWithTimeout:30.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+// TODO: figure out why this test fails when run together, but passes individually
```

Comment:
Most likely because the API Config might have got changed some other test cases running in different order. So, you might have to reset it before the test case runs. Will show you a sample.